# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-masking/clip-path/clip-path-geometryBox-2.html.ini
# Commit: 63f54f50c32d
# Full Hash: 63f54f50c32dcbfa01321485c0cafad0c5bbfbba
# Author: Jeff Muizelaar <jmuizelaar@mozilla.com>
# Date: 2022-02-19 09:33:23
# Regressor Bug: 1756197
# File Overlap Count: 1
# Description:
#   Bug 1756197. Avoid clipping if we're just going to fill the path. r=gfx-reviewers,nical
#   
#   This avoids having to rasterize the path twice. Once for the clip path and once for
#   the fill. It will also give us correct antialiasing on the edges.
#   
# ==============================================================================

diff -r 0886d48d6a83 -r 63f54f50c32d testing/web-platform/meta/css/css-masking/clip-path/clip-path-geometryBox-2.html.ini
--- a/testing/web-platform/meta/css/css-masking/clip-path/clip-path-geometryBox-2.html.ini	Fri Feb 18 22:04:43 2022 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,2 +0,0 @@
-[clip-path-geometryBox-2.html]
-  expected: FAIL
